### PR TITLE
fix: Free view bindings in dialogs

### PR DESCRIPTION
**Pull request by Isira-Seneviratne**
Allow dialog view bindings to be garbage collected when the dialog is destroyed.

### CODE REVIEW

File: app/src/main/java/com/github/libretube/ui/sheets/IntentChooserSheet.kt [hunk 1]
```diff
@@ -26,6 +26,7 @@ class IntentChooserSheet(
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        val binding = binding
```
Comment (Bnyro) [inline app/src/main/java/com/github/libretube/ui/sheets/IntentChooserSheet.kt:29]: What exactly is this doing?

Comment (Isira-Seneviratne) [reply app/src/main/java/com/github/libretube/ui/sheets/IntentChooserSheet.kt:29]: It's calling the non-null getter method and holding the value.

Comment (Bnyro) [reply app/src/main/java/com/github/libretube/ui/sheets/IntentChooserSheet.kt:29]: Interesting, never seen that before.

**Review by Bnyro — APPROVED** [review]

File: app/src/main/java/com/github/libretube/ui/sheets/IntentChooserSheet.kt (hunk 1 above)
Comment (Bnyro) [reply app/src/main/java/com/github/libretube/ui/sheets/IntentChooserSheet.kt:29]: But makes sense, yeah.

**Pull request merged**